CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Various fixes [Theodor]
```

Comment:
I guess this could be more informative, maybe "Various build fixes to support nVidia TX2 build" or something like that.